test/metabase/api/search_test.clj

```diff
@@ -551,42 +551,42 @@
 (deftest indexed-entity-test
   (testing "Should search indexed entities"
     (mt/dataset airports
-      (let [query (mt/mbql-query municipality)]
-        (mt/with-temp [Card model {:dataset       true
-                                   :dataset_query query}]
-          (let [model-index (model-index/create
-                             (mt/$ids {:model-id   (:id model)
-                                       :pk-ref     $municipality.id
-                                       :value-ref  $municipality.name
-                                       :creator-id (mt/user->id :rasta)}))
-                relevant    (comp (filter (comp #{(:id model)} :model_id))
-                                  (filter (comp #{"indexed-entity"} :model)))
-                search!     (fn [search-term]
-                              (:data (make-search-request :crowberto [:q search-term])))]
-            (model-index/add-values! model-index)
-
-            (is (= #{"Dallas-Fort Worth" "Fort Lauderdale" "Fort Myers"
-                     "Fort Worth" "Fort Smith" "Fort Wayne"}
-                   (into #{} (comp relevant (map :name)) (search! "fort"))))
-
-            (testing "Sandboxed users do not see indexed entities in search"
-              (with-redefs [premium-features/sandboxed-or-impersonated-user? (constantly true)]
-                (is (= #{}
-                       (into #{} (comp relevant (map :name)) (search! "fort"))))))
-
-            (let [normalize (fn [x] (-> x (update :pk_ref mbql.normalize/normalize)))]
-              (is (=? {"Rome"   {:pk_ref         (mt/$ids $municipality.id)
-                                 :name           "Rome"
-                                 :model_id       (:id model)
-                                 :model_name     (:name model)
-                                 :model_index_id #hawk/malli :int}
-                       "Tromsø" {:pk_ref         (mt/$ids $municipality.id)
-                                 :name           "Tromsø"
-                                 :model_id       (:id model)
-                                 :model_name     (:name model)
-                                 :model_index_id #hawk/malli :int}}
-                      (into {} (comp relevant (map (juxt :name normalize)))
-                            (search! "rom")))))))))))
+                (let [query (mt/mbql-query municipality)]
```

Comment:
`mt/dataset` is a special macro that only indents two spaces. Unfortunately calva doesn't know this so you have to do it by hand.